site/content/posts/2023-09-18-multiple-idps-and-identity-transformations.md

```diff
@@ -0,0 +1,129 @@
+*Photo from [Unsplash](https://unsplash.com/photos/fcmxfsAgxqU)*
+
+Pinniped's v0.26.0 release includes several significant new features described below.
+Additionally, the release includes several dependency updates and fixes.
+See the [release notes](https://github.com/vmware-tanzu/pinniped/releases/tag/v0.26.0) for more details.
+
+## Background: Identity provider resources
+
+In Kubernetes, a user identity is a username string and a list of group name strings.
+
+Each OIDCIdentityProvider, LDAPIdentityProvider, or ActiveDirectoryIdentityProvider configures how to use a specific
+protocol (e.g. OIDC or LDAP) to authenticate users and to extract their identities (usernames and group names) for
+Kubernetes from those external identity providers.
+
+Prior to this release, the Pinniped Supervisor would only allow the use of a single OIDCIdentityProvider, LDAPIdentityProvider,
+or ActiveDirectoryIdentityProvider at a time. With this new release, you can now use multiple simultaneous identity
+providers from which to draw user identities.
+
+## New: Multiple FederationDomains
+
+The Pinniped Supervisor offers a custom resource called FederationDomain.
+Any Kubernetes cluster may trust a Supervisor's FederationDomain to provide authentication services for the cluster.
+Each FederationDomain defines which users may authenticate into those clusters, and how those users go about authenticating.
+So, for a given FederationDomain, the *same users* may authenticate into *all* the clusters which trust that FederationDomain.
+
+Once authenticated, each cluster may use different authorization policies (e.g. Kubernetes RBAC rules) to determine the
+capabilities of each user. Pinniped provides user authentication, while still allowing you to choose your preferred
+system for user authorization on each cluster.
+
+You may configure multiple FederationDomains in the Pinniped Supervisor. Each must have a unique URL called an "issuer URL".
+These URLs can be unique by using different hostnames (a form of virtual hosting) or they can simply have different paths on the same host.
+
+When a user authenticates to a FederationDomain, they start a single sign-on session for that FederationDomain only.
+The user may use all clusters which trust that FederationDomain for the rest of the day without being prompted to
+authenticate again. Behind the scenes, the Supervisor is constantly checking with the external identity provider to ensure
+that the user should be allowed to continue their ongoing session.
+
+Prior to this release, there was no way to configure each FederationDomain to be meaningfully different from the others
+within a single Pinniped Supervisor. However, starting with this release, you can now configure each FederationDomain
+to allow a different pool of users. (See below for more details.) This makes it possible to use multiple FederationDomains
+in a single Pinniped Supervisor.
```

Comment:
`in a single Pinniped Supervisor.`. This provides multiple groupings of multiple identity providers for multiple clusters via a single Pinniped Supervisor!

A little bit of cheerleading vibe again, I know, but I think it would help to very specifically tell a user (and a search engine) why this is valuable.